sql/core/src/main/scala/org/apache/spark/sql/execution/objects.scala

```diff
@@ -451,8 +451,6 @@ case class FlatMapGroupsInRExec(
 
   override def outputPartitioning: Partitioning = child.outputPartitioning
 
-  override def producedAttributes: AttributeSet = AttributeSet(outputObjAttr)
```

Comment:
Isn't `outputObjAttr` produced by `FlatMapGroupsInRExec`?

Reply:
This is a no-op override, it's the same as `ObjectProducerExec.producedAttributes`